src/main/java/org/mqttbee/api/mqtt/MqttSslConfig.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 The MQTT Bee project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mqttbee.api.mqtt;
+
+import org.mqttbee.annotations.NotNull;
+import org.mqttbee.annotations.Nullable;
+
+import java.security.KeyStore;
+
+public class MqttSslConfig {
```

Comment:
This looks like a mixture of builder and config entity.
Imho the MqttClientSslData should be renamed to MqttClientSslConfig and this class should be converted into the MqttClientSslConfigBuilder

Reply:
I agree, is better, but I will not have time to do this in the next 2 weeks

Reply:
No worries, somebody else can do this